lib/reference/properties.js

```diff
@@ -1,8 +1,18 @@
 'use strict';
 
-module.exports = {
-	margin: new Set(['margin-top', 'margin-bottom', 'margin-left', 'margin-right']),
-	padding: new Set(['padding-top', 'padding-bottom', 'padding-left', 'padding-right']),
+/** @type {import('stylelint').LonghandSubPropertiesOfShorthandProperties} */
+const longhandSubPropertiesOfShorthandProperties = {
+	/* eslint sort-keys: 'error' */
```

Comment:
[note] I've sorted properties alphabetically for findability (the diff increases, though).